erp/models.py

```diff
@@ -1715,3 +1715,7 @@ def get_labels_display(self):
     def get_accueil_audiodescription_text(self):
         equipment_to_text = {k: str(v) for k, v in schema.AUDIODESCRIPTION_CHOICES}
         return ",".join([equipment_to_text.get(equipment) for equipment in self.accueil_audiodescription])
+
+    def get_entree_dispositif_appel_type_text(self):
+        equipment_to_text = {k: str(v) for k, v in schema.DISPOSITIFS_APPEL_CHOICES}
+        return [equipment_to_text.get(equipment) for equipment in self.entree_dispositif_appel_type]
```

Comment:
```suggestion
        return [equipment_to_text.get(equipment) for equipment in (self.entree_dispositif_appel_type or []])
```
Or an early return. 

We have a ~23k ERPs with `None` as value, which would raises a TypeError (NoneType not iterable).

Same for above helpers.